docs/02-app/02-api-reference/05-next-config-js/index.mdx

```diff
@@ -5,7 +5,7 @@ description: Learn how to configure your application with next.config.js.
 
 {/* The content of this doc is shared between the app and pages router. You can use the `<PagesOnly>Content</PagesOnly>` component to add content that is specific to the Pages Router. Any shared content should not be wrapped in a component. */}
 
-Next.js can be configured through a `next.config.js` file in the root of your project directory (for example, by `package.json`).
+Next.js can be configured through a `next.config.js` file in the root of your project directory (for example, by `package.json`) with a default export.
```

Comment:
Oh yes the examples look good to me, just wanted to state as text that it needs a default export.

Reply:
Oh yeah, I forgot this PR is also updating the docs 😅

Lets just change it to only update the docs, thanks!